editor/components/default-block-appender/index.js

```diff
@@ -27,15 +27,16 @@ export function DefaultBlockAppender( { isLocked, isVisible, onAppend, showPromp
 	return (
 		<div className="editor-default-block-appender">
 			<BlockDropZone />
-			<input
-				className="editor-default-block-appender__content"
-				type="text"
-				readOnly
-				onFocus={ onAppend }
-				onClick={ onAppend }
-				onKeyDown={ onAppend }
-				value={ showPrompt ? __( 'Write your story' ) : '' }
-			/>
+			<div className="editor-default-block-appender__content">
```

Comment:
There are failing snapshot tests because of the change in markup. Is this nesting really necessary though? Could we apply `padding-top` to the parent `.editor-default-block-appender` instead?

Reply:
Great catch. I could swear it was necessary to make this change, so the hovered style matched what the end paragraph would be, but a later change must've made it unnecessary because reverting this fixes it.